CONTRIBUTING.md

```diff
@@ -83,101 +100,91 @@ pytest -rs tests/integrations/flask/  # Replace "flask" with the specific integr
-
-(only relevant for Sentry employees)
+## Adding a New Integration
 
-Prerequisites:
-
-- All the changes that should be release must be in `master` branch.
-- Every commit should follow the [Commit Message Format](https://develop.sentry.dev/commit-messages/#commit-message-format) convention.
-- CHANGELOG.md is updated automatically. No human intervention necessary.
-
-Manual Process:
-
-- On GitHub in the `sentry-python` repository go to "Actions" select the "Release" workflow.
-- Click on "Run workflow" on the right side, make sure the `master` branch is selected.
-- Set "Version to release" input field. Here you decide if it is a major, minor or patch release. (See "Versioning Policy" below)
-- Click "Run Workflow"
-
-This will trigger [Craft](https://github.com/getsentry/craft) to prepare everything needed for a release. (For more information see [craft prepare](https://github.com/getsentry/craft#craft-prepare-preparing-a-new-release)) At the end of this process a release issue is created in the [Publish](https://github.com/getsentry/publish) repository. (Example release issue: https://github.com/getsentry/publish/issues/815)
+1. Write the integration.
 
-Now one of the persons with release privileges (most probably your engineering manager) will review this Issue and then add the `accepted` label to the issue.
+   - Instrument all application instances by default. Prefer global signals/patches instead of configuring a specific instance. Don't make the user pass anything to your integration for anything to work. Aim for zero configuration.
 
-There are always two persons involved in a release.
+   - Everybody monkeypatches. That means:
 
-If you are in a hurry and the release should be out immediatly there is a Slack channel called `#proj-release-approval` where you can see your release issue and where you can ping people to please have a look immediatly.
+     - Make sure to think about conflicts with other monkeypatches when monkeypatching.
 
-When the release issue is labeled `accepted` [Craft](https://github.com/getsentry/craft) is triggered again to publish the release to all the right platforms. (See [craft publish](https://github.com/getsentry/craft#craft-publish-publishing-the-release) for more information). At the end of this process the release issue on GitHub will be closed and the release is completed! Congratulations!
+     - You don't need to feel bad about it.
 
-There is a sequence diagram visualizing all this in the [README.md](https://github.com/getsentry/publish) of the `Publish` repository.
+   - Avoid modifying the hub, registering a new client or the like. The user drives the client, and the client owns integrations.
 
-### Versioning Policy
+   - Allow the user to disable the integration by changing the client. Check `Hub.current.get_integration(MyIntegration)` from within your signal handlers to see if your integration is still active before you do anything impactful (such as sending an event).
```

Comment:
```suggestion
   - Allow the user to turn-off the integration by changing the client. Check `Hub.current.get_integration(MyIntegration)` from within your signal handlers to see if your integration is still active before you do anything impactful (such as sending an event).
```

Reply:
@cleptric No hyphen is needed in "turn-off." The correct spelling is "turn off."

Although, I think using "disable" here is more appropriate, as @sentrivana did originally.

Reply:
Will apply this as `turn off` (just a regular phrasal verb, no dashes)